src/foremast/securitygroup/create_securitygroup.py

```diff
@@ -182,6 +185,7 @@ def create_security_group(self):
 
         try:
             ingress = self.properties['security_group']['ingress']
+            ingress.update(DEFAULT_SECURITYGROUP_RULES)
```

Comment:
Normally I don't like to bunch more than one operation in a `try:except`, but a `dict.update()` is a fairly safe operation and will most likely raise a `ValueError` if it fails at this point. The other option would be to move it outside of the `try:except` and handle any exceptions if they come up in the future.